src/amo/api/index.js

```diff
@@ -166,12 +165,9 @@ export function callApi({
   adjustedEndpoint = adjustedEndpoint.endsWith('/')
     ? adjustedEndpoint
     : `${adjustedEndpoint}/`;
-  let apiURL = `${config.get('apiHost')}${adjustedEndpoint}${queryString}`;
-  if (_config.get('server')) {
-    _log.debug('Encoding `apiURL` in UTF8 before fetch().');
-    // Workaround for https://github.com/bitinn/node-fetch/issues/245
-    apiURL = utf8.encode(apiURL);
-  }
+  const apiURL = encodeURI(
```

Comment:
I don't understand why we are now encoding on both the client and server, whereas before we were only encoding on the server.